evals/ground_truth.jsonl

```diff
@@ -0,0 +1,24 @@
+{"question": "Could you provide details on the insurance coverage Contoso offers for work-related injuries? Additionally, I'd like to know more about the overall benefits package available to employees.", "truth": "Contoso offers Workers' Compensation Insurance coverage through Northwind Health for work-related injuries or illnesses [Northwind_Standard_Benefits_Details.pdf#page=101][Northwind_Health_Plus_Benefits_Details.pdf#page=106]. This coverage includes medical care, wage replacement, vocational rehabilitation, and death benefits [Northwind_Standard_Benefits_Details.pdf#page=101]. Employees should report any injuries or illnesses to their supervisor as soon as possible and ensure that the appropriate paperwork is on file to be eligible for Workers' Compensation Insurance coverage [Northwind_Health_Plus_Benefits_Details.pdf#page=107]. As for the overall benefits package, please provide more specific details or refer to the employee handbook."}
```

Comment:
How did you generate this?

Reply:
This looks like the ground truth generated from the azure-ai-eval simulator. I didn't think this was as useful, so as long as RAGAS looks good, I'll remove this file and the non-RAGAS generation.